# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libxft/hotfix.patch
# Copyright (C) 2013 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libXft-2.3.1/src/xftglyphs.c.vanilla	2013-12-19 16:47:16.658813666 +0100
+++ libXft-2.3.1/src/xftglyphs.c	2013-12-19 16:47:51.222814955 +0100
@@ -21,10 +21,10 @@
  */
 
 #include "xftint.h"
-#include <freetype/ftoutln.h>
-#include <freetype/ftlcdfil.h>
+#include <ftoutln.h>
+#include <ftlcdfil.h>
 
-#include <freetype/ftsynth.h>
+#include <ftsynth.h>
 
 /*
  * Validate the memory info for a font
